index.js

```diff
@@ -14,11 +14,13 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell
-//. and PureScript. It's stricter and more opinionated than [Ramda][],
-//. and provides a similar suite of functions.
+//. Sanctuary is a functional programming library inspired by Haskell and PureScript.
+//. It provides a similar suite of functions as [Ramda](https://github.com/ramda/ramda), but in a stricter and more
+//. opinionated way, ensuring errors are properly handled.
+//. Sanctuary makes it easy to build a program up from smaller types and functions
+//. and makes it possible to write safe code without constant null-checks.
+//. It uses and provides the two [Fantasy Land](https://github.com/fantasyland/fantasy-land)-compatable monads Maybe and Either.
```

Comment:
Let's replace `[Fantasy Land](https://github.com/fantasyland/fantasy-land)` with `[Fantasy Land][]`, and add this link reference at the bottom of the file:

```javascript
//. [Fantasy Land]:     v:fantasyland/fantasy-land
```

See #324 if you'd like to know the significance of the `v:` above.